packages/angular_devkit/core/src/jobs/README.md

```diff
@@ -0,0 +1,483 @@
+# Description
+
+Jobs is the Angular DevKit subsystem for scheduling and running generic functions with clearly 
+typed inputs and outputs. A `Job` instance is code (ie. a function) that can be scheduled,
+synchronized and schedule other jobs. Those jobs can be run in parallel or follow an execution 
+strategy. They can also be extended to allow for transformations, combination, and refactoring 
```

Comment:
perhaps:  ... can be scheduled, be synchronized with other jobs, and start other jobs.